### PR TITLE
Priyanshu-Bootcamp

### PR DESCRIPTION
**Pull request by hydroxyhelium**
Here is a screenshot of the training. It achieved a final testing accuracy of 76.190% on the testing set. 
Screenshot 2022-10-13 at 10 23 21 PM

### CODE REVIEW

**Review by mikashaw — APPROVED** [review]
Hey Priyanshu, congrats on completing the CV bootcamp! Your code looks great, I really like how object-oriented it is (that will be super helpful when developing in CV), based on the loss plots, it looks like the model may be slightly be overfitting at around 5 epochs, but thats quite minor. Great job overall and welcome to CV! We're currently doing a team stand down for midterms but our next official CV meeting will be the 27th. Onboarding documents can be found here: https://uwarg-docs.atlassian.net/wiki/spaces/CV/pages/2060353537/CV+Onboarding. At the bottom of this document https://uwarg-docs.atlassian.net/wiki/spaces/CV/pages/2108882951/2022-09-27 you'll find a list of projects that are ongoing this term. Please select one and write your top 3 in the project-selection CV channel on discord and myself or Aydan will get back to you, also, if you have any questions message me or aydan on discord and we'll be happy to help!
